### PR TITLE
Provide Complete Automation Example

### PR DESCRIPTION
**Pull request by OhneNik**
Hey,

Just went through the issues to get the pv surplus charging to work with an automation. Just an update to the readme, could be useful to some people ¯\_(ツ)_/¯

### CODE REVIEW

**Comment by sanchosk** [comment]
This looks very interesting.
So just by providing the info about excess solar (e.g. the number in the automation will be negative meaning feeding the grid) it will start charge accordingly?
Or am I missing something?

**Comment by OhneNik** [comment]
@sanchosk basically yes. Just take a look at the paragraph where I added the full example, it‘s all explained there.

File: README.md [hunk 1]
```diff
@@ -309,8 +309,19 @@ See template example below for how to continuously update `ids`
 
 #### Automation example
 ```
-      payload: '{{''{"pGrid": ''}}{{states(''sensor.power_use'')}}{{'', "pPv": ''}}{{states(''sensor.solar_production'')}}{{'',
-        "pAkku":0}''}}'
+   alias: Go-E Überschussladen
```
Comment (syssi) [inline README.md:312]: Could you provide an alias and description in english?

File: README.md [hunk 2]
```diff
@@ -309,8 +309,19 @@ See template example below for how to continuously update `ids`
...
+   description: ""
+   trigger:  
+      - platform: time_pattern
```
Comment (syssi) [inline README.md:315]: Home Assistant recommends indents using 2 spaces (https://developers.home-assistant.io/docs/documenting/yaml-style-guide/). Could you change the indents? Thanks in advance!

**Comment by OhneNik** [comment]
@syssi sorry, wasn‘t using a real editor there while the dog slept on my lap.

**Review by syssi — APPROVED** [review]

**Comment by syssi** [comment]
Thanks for your great first contribution! If you see room for improvements feel free to create an issue or prepare another pull request! This PR is ready to go!

**Pull request merged**